examples/hello-world/Makefile.c64.defines

```diff
@@ -0,0 +1,2 @@
+DEFINES = WITH_CLIENT,WITH_DNS,WITH_PFS,WITH_LOGGING
+CONTIKI_WITH_IPV4 = 1
```

Comment:
If you check out the recipe `savedefines` then you'll notice that Makefile.<target>.defines are not general containers for Makefile variable settings but are rather intended to contain exactly one line saying `DEFINES = ...`. From that perspective your approach to configure IPv4 doesn't seem valid to me.

As already said several times Contiki unfortunately presumes that IPv4 vs. IPv6 is purely a property of the example but but a property of the example-target tupel. I personally consider this a bug.